src/packages/dictionary/dashboards/dictionary/dashboard-localization-dictionary.element.ts

```diff
@@ -124,40 +112,25 @@ export class UmbDashboardTranslationDictionaryElement extends UmbLitElement {
 
 	#filter(e: { target: HTMLInputElement }) {
 		this._tableItemsFiltered = e.target.value
-			? this.#tableItems.filter((t) => t.id.includes(e.target.value))
+			? this.#tableItems.filter((t) => t.id.toLocaleLowerCase().includes(e.target.value.toLocaleLowerCase()))
```

Comment:
This is only an optimization, but consider running any functions outside of the filter method first so you don't have to run them inside a loop for each iteration. Doing that will optimize the execution moving from O(n) to O(1).

In this case, you could store the value of `e.target.value.toLocaleLowerCase()` outside the filter call in a local variable and then refer to that when comparing with `includes()`.